setup.py

```diff
@@ -15,12 +22,14 @@
     url = 'https://bitbucket.org/kmike/yandex-maps/',
 
     description = 'Yandex.Maps API python wrapper with optional django integration.',
-    long_description = open('README.rst').read().decode('utf8') + open('CHANGES.rst').read().decode('utf8'),
+    long_description = open('README.rst').read() + open('CHANGES.rst').read(),
```

Comment:
This will fail in Python 3 on systems with non-UTF8 locale because 'open' uses `locale.getpreferredencoding()` encoding by default